### DISCUSSION TITLE
The try_get function on MockRow unwraps

### DISCUSSION THREAD

**Discussion by cemoktra** [SeaQL/sea-orm#626] [Q&A]
## Description

While a real implemention of try_get returns an error, the MockRow unwraps on not found columns. This makes testing a bit harder:
```rust
impl MockRow {
    /// Try to get the values of a [MockRow] and fail gracefully on error
    pub fn try_get<T>(&self, col: &str) -> Result<T, DbErr>
    where
        T: ValueType,
    {
        T::try_from(self.values.get(col).unwrap().clone()).map_err(|e| DbErr::Query(e.to_string()))
    }
```

## Steps to Reproduce

1. Have a struct with custom `FromQueryResult`
2. Have JSON_EXTRACT columns that are optional

### Expected Behavior

It should return `DbErr`

### Actual Behavior

It unwraps and therefore panicks

### Reproduces How Often

Always

## Versions

0.6

**Comment by billy1624** [comment]
Hey @cemoktra, thanks for the bug report! I'm still not sure how to reproduce it. Could you please provide a minimal reproducible example? Thanks!!

**Reply by cemoktra** [reply]
Well i guess i had some columns missing in my mapped mock, and was expecting it to return an Error instead of unwrapping
